docs/administration-guide/13-embedding.md

```diff
@@ -116,25 +116,27 @@ Dashboards are a fixed aspect ratio, so if you'd like to ensure they're automati
 <iframe src="http://metabase.example.com/embed/dashboard/TOKEN" onload="iFrameResize({}, this)"></iframe>
 ```
 
-### Additional parameters
+## Additional parameters
 
 To change the look and feel of the embed, you can add additional parameters to the embedding URL:
 
 - **bordered**: true or false. Adds a visible border to the embed.
 - **titled**: true or false. Adds or removes the title to the embedding.
 - **theme**: null or night. Shows the embedded iframe in dark mode.
 
-For example: http://yourmetabaseurl.com/embed/dashboard/a_very_huge_hashed_url#theme=night&hide_parameters=category&titled=true&bordered=false
+For example: 
 
-### Reference applications
+```
+http://yourmetabaseurl.com/embed/dashboard/a_very_huge_hashed_url#theme=night&hide_parameters=category&titled=true&bordered=false
+```
 
-To see concrete examples of how to embed Metabase in applications under a number of common frameworks, check out our [reference implementations](https://github.com/metabase/embedding-reference-apps) on Github.
+These settings can also be changed under **Style** when previewing the embedded question or dashboard and its code in Metabase.
 
-## Premium embedding
+## Reference applications
 
-If you'd like to embed Metabase dashboards or charts in your application without the "Powered by Metabase" attribution, you can purchase premium embedding from the Metabase store. [Find out more here](https://store.metabase.com/product/embedding).
+To see concrete examples of how to embed Metabase in applications under a number of common frameworks, check out our [reference implementations](https://github.com/metabase/embedding-reference-apps) on Github.
 
-To learn more about embedding, check out these articles:
+## Further reading
```

Comment:
Maybe instead we should link to the entire learn track on embedding:

https://www.metabase.com/learn/embedding